Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/qrcode/DccTicketingQrCodeExtractor.kt

```diff
@@ -29,6 +29,7 @@ class DccTicketingQrCodeExtractor @Inject constructor(
         }
     }
 
+    @Suppress("UselessCallOnNotNull")
```

Comment:
Can we remove it if `null` is not expected anymore ?

Reply:
it will crash here if the value was null (not while parsing) and then expected error code is not showing